types/errors/errors.go

```diff
@@ -166,8 +166,8 @@ var (
 	// ErrORMUniqueConstraint defines an error when a value already exists at a given key
 	ErrORMUniqueConstraint = Register(ormCodespace, 44, "unique constraint violation")
 
-	// ErrORMEmptyModel defines an error when an empty model is provided for building a table
-	ErrORMEmptyModel = Register(ormCodespace, 45, "invalid argument")
+	// ErrORMInvalidArgument defines an error when an invalid argument is provided as part of ORM functions
+	ErrORMInvalidArgument = Register(ormCodespace, 45, "invalid argument")
```

Comment:
this is based on this previous discussion: https://github.com/cosmos/cosmos-sdk/pull/10035#discussion_r708322730

Reply:
I really disagree with this approach. types should not need to know about math and orm. Let's please keep these in x/group. They can be public there not types/

Reply:
It can be in a separate PR but please let's move them

Reply:
In this case, we might want to move them to a subpackage x/group/errors for instance because that might cause an import cycle between group and orm, wdyt?